controller/volume_controller.go

```diff
@@ -1585,6 +1585,15 @@ func (vc *VolumeController) ReconcileVolumeState(v *longhorn.Volume, es map[stri
 					continue
 				}
 
+				// The node where the replica is located is deleted, the replica can be directly marked as failed.
+				if _, err := vc.ds.GetNodeRO(r.Spec.NodeID); err != nil && apierrors.IsNotFound(err) {
```

Comment:
Should also handle other errors.

Reply:
Updated to skip replica if a non-`NotFound` error is encountered. Thank you.